build-and-push-to-dockerhub.sh

```diff
@@ -21,14 +31,8 @@ platforms=linux/arm64,linux/amd64
 # build and push images
 for docker_tag in "${@:2}"
 do
+    # Check that valid tag is supplied. If not, fail
```

Comment:
This comment is no longer valid here as the check was moved to lines 9-17.

Reply:
good spot